# 这怎么查错？

这报错都不报错在程序哪一行怎么查错啊

TypeError                                 Traceback (most recent call last)
<mercury-input-1-D83645B4CC5F497788910C8C47D8FCB1> in <module>()
    115                                             accounts=accounts, preload_data=_QUARTZ_PRELOAD_DATA,
    116                                             display=True, return_quartz_data=True,
--> 117                                             threaded=quartz_createVar.get('threaded', True))
    118     _QUARTZ_CACHE['start'] = start
    119     _QUARTZ_CACHE['end'] = end

/home/ipython/anaconda/lib/python2.7/site-packages/mercuryq-quartz.egg/quartz/backtest.pyc in backtest(start, end, benchmark, universe, capital_base, initialize, handle_data, post_trading_day, commission, slippage, refresh_rate, freq, security_base, security_cost, max_history_window, accounts, threaded, display, preload_data, *args, **kwargs)

/home/ipython/anaconda/lib/python2.7/site-packages/mercuryq-quartz.egg/quartz/backtest.pyc in bt_algorithm(clock, trading_scheduler, trading_agent, threaded)

/home/ipython/anaconda/lib/python2.7/site-packages/mercuryq-quartz.egg/quartz/trading_agent.pyc in handle_data(self)

<mercury-input-1-D83645B4CC5F497788910C8C47D8FCB1> in handle_data(context)
     40     amount = 100
     41 
---> 42     history_data = context.history(symbol=symbol, attribute=['closePrice', 'openPrice', 'lowPrice', 'highPrice'], time_range=30, freq='1d')[symbol]
     43     upper_band, middle_band, lower_band = talib.BBANDS(history_data['closePrice'].apply(float).values, timeperiod = 30, nbdevup = 2, nbdevdn = 2)
     44     pre_close_price = history_data['closePrice'][-1]

TypeError: unhashable type: 'list'